### ISSUE TITLE
CodeBuild concurrent builds limit

### ISSUE THREAD

**Issue by pako-grape** [hashicorp/terraform-provider-aws#17942]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

There is the possibility to set concurrent builds limit in an AWS CodeBuild project (under Project Configuration):
![image](https://user-images.githubusercontent.com/13254915/110087355-d4df4a80-7d93-11eb-9dd4-bcda186f5e14.png)

It would be very cool to control this configuration part in terraform

### New or Affected Resource(s)

* aws_codebuild_project

### Potential Terraform Configuration

```hcl
resource "aws_codebuild_project" "example" {
    concurrent_builds_limit = 1
}
```

**Comment by ewbankkit** [comment]
CodeBuild concurrent build support was made available in [AWS SDK v1.37.12](https://github.com/aws/aws-sdk-go/releases/tag/v1.37.12) but has not yet been implemented in the [`aws_codebuild_project`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/codebuild_project) resource.

**Comment by andyt10** [comment]
If this is already in the version of AWS SDK used by the provider, then I imagine this would be a relatively minor change...and a good thing for a first time contributor? I'd be interested in doing this, if so. :)
